package.json

```diff
@@ -15,6 +15,7 @@
     "@babel/runtime": "^7.0.0-beta.38",
     "babel-runtime": "^6.26.0",
     "bcrypt": "^1.0.3",
+    "formik": "^1.4.2",
```

Comment:
If we're not using this yet, let's uninstall it to make sure we don't have unused stuff in the bundle